src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -0,0 +1,259 @@
+              </Tab>
+            )
+          })}
+        </Tabs>
+      )}
+    </>
+  )
+}
+
+const BinaryDownloadsSection = ({
+  downloadsByOS,
+  os,
+  prettyOSName,
+  selectedRelease,
+}) => {
+  const { name, version } = selectedRelease
+  return (
+    <>
+      <Heading
+        {...SHARED_HEADING_LEVEL_3_PROPS}
+        slug={`binary-download-for-${prettyOSName}`}
+      >
+        Binary download for {prettyOSName}
+      </Heading>
+      {Object.keys(downloadsByOS[os]).map((arch) => (
+        <Card className={s.textAndLinkCard} elevation="base" key={arch}>
+          <div className={s.textAndLinkCardTextContainer}>
+            <Text
+              className={s.textAndLinkCardLabel}
+              size={200}
+              weight="semibold"
+            >
+              {arch.toUpperCase()}
+            </Text>
+            <Text
+              className={s.textAndLinkCardVersionLabel}
+              size={200}
+              weight="regular"
+            >
+              Version: {version}
+            </Text>
+          </div>
+          <DownloadStandaloneLink
+            ariaLabel={`download ${name} version ${version} for ${prettyOSName}, architecture ${arch}`}
+            href={downloadsByOS[os][arch]}
+          />
+        </Card>
+      ))}
+    </>
+  )
+}
+
+const ChangelogSection = ({ selectedRelease }) => {
+  const { name, version } = selectedRelease
+  return (
+    <>
+      <Heading {...SHARED_HEADING_LEVEL_3_PROPS} slug="release-information">
+        Release information
+      </Heading>
+      <Card className={s.textAndLinkCard} elevation="base">
+        <div className={s.textAndLinkCardTextContainer}>
+          <Text className={s.textAndLinkCardLabel} size={200} weight="semibold">
+            Changelog
+          </Text>
+          <Text
+            className={s.textAndLinkCardVersionLabel}
+            size={200}
+            weight="regular"
+          >
+            Version: {version}
+          </Text>
+        </div>
+        <StandaloneLink
+          ariaLabel={`${name} version ${version} changelog`}
+          href={`https://github.com/hashicorp/waypoint/blob/v${version}/CHANGELOG.md`}
+          icon={<IconExternalLink16 />}
+          iconPosition="trailing"
+          openInNewTab
+          text="GitHub"
+          textSize={200}
+        />
+      </Card>
+    </>
+  )
+}
+
+const NotesSection = ({ selectedRelease }) => {
+  const { name, shasums, shasums_signature, version } = selectedRelease
+  return (
+    <>
+      <Heading
+        {...SHARED_HEADING_LEVEL_3_PROPS}
+        className={classNames(
+          SHARED_HEADING_LEVEL_3_PROPS.className,
+          s.specialSubHeading
+        )}
+        slug="notes"
+      >
+        Notes
+      </Heading>
+      <Text size={200}>
+        You can find the{' '}
+        <InlineLink
+          href={`https://releases.hashicorp.com/${name}/${version}/${shasums}`}
+          text={`SHA256 checksums for Waypoint ${version}`}
```

Comment:
Is the plan to address some of the hardcoded product name usages later on?

Reply:
Oops! Great catch. This should already be using `name` instead of `Waypoint`. Will fix it!

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/130/commits/0567f6247f916b98c25af7c7bb81e863cdd7214c